src/main/java/com/github/fishio/CollisionMask.java

```diff
@@ -10,7 +12,7 @@
  */
 public class CollisionMask implements ICollisionArea {
 	
-	private static final boolean PIXEL_PERFECT_COLLISIONS = true;
+	private Settings settings = Settings.getInstance();
```

Comment:
Do you need to keep a reference to the settings? Especially since this class only requests a setting in one place, you could simply replace settings with Settings.getInstance() there.

<organisation